.gitignore

```diff
@@ -9,3 +9,8 @@
 *.jar
 build.xml
 /nbproject/
+
+# Intellij 
+*.iml
+.idea/
+/out/

```

Comment:
Please change `/out/` for `out/`.  The excluded directories already specified in this file contain a mistake, because directories shouldn't start with a `/`. The ending `/` is required though. 

Please also remove the front `/` for the other directories in the same time. 

```
/nbproject/private/
/build/
/dist/
/nbproject/
```

Thanks!